bin/3llo

```diff
@@ -10,10 +10,29 @@ $container.register(:api_client, Tr3llo::HTTP::Client)
 prompt = TTY::Prompt.new
 $container.register(:interface, Tr3llo::Interface.new(prompt, $stdout))
 
+def print_help(flavour)
+  Tr3llo::Presenter::HelpPresenter
+    .new($container.resolve(:interface), flavour)
+    .print!
+end
+
+unless ARGV.empty?
+  if ARGV[0] =~ /^-h|--help$/
```

Comment:
We could maybe making it more explicit, since we aren't going to support multiple switches:

```rb
if ARGV.size == 1 && (ARGV[0] == "-h" || ARGV[0] == "--help")
  ...
end
```